server/src/modules/models/video/request.js

```diff
@@ -4,6 +4,7 @@ const schema = Joi.object().keys({
   _id: Joi.string().optional(),
   title: Joi.string().min(3).max(30).required(),
   fileName: Joi.string().min(3).max(30).required(),
+  originalName: Joi.string().min(3).max(30).required(),
```

Comment:
not sure why we need this, but that's fine for now.

Reply:
Because it is required property in schema. If I put `originalName` in request body it shows property is not allowed as it was not present there previously. That is why I put it here.